README.md

```diff
@@ -27,6 +27,10 @@ mainline is written using Vala and GTK3. Source code and binaries are available
 	make
 	sudo make install
 
+### Run
+
+Run `mainline` for the command line version, and `mainline-gtk` for the grpahical interface.
```

Comment:
```suggestion
Run `mainline` for the command line version, and `mainline-gtk` for the graphical interface.
```

Reply:
Found a small typo, other than that it's always good to have some documentation :+1: